rustls-libssl/src/lib.rs

```diff
@@ -282,6 +286,26 @@ impl SslContext {
     fn set_alpn_offer(&mut self, alpn: Vec<Vec<u8>>) {
         self.alpn = alpn;
     }
+
+    fn stage_certificate_end(&mut self, end: CertificateDer<'static>) {
```

Comment:
nit: would mildly prefer this fn to be named `stage_certificate_end_entity`. Ditto for the `Ssl` version.